libr/include/r_util/r_str.h

```diff
@@ -200,6 +206,7 @@ R_API const char *r_str_rsep(const char *base, const char *p, const char *sep);
 R_API char *r_str_donut(int size);
 R_API char *r_str_version(const char *program);
 
+R_API size_t r_charset_encode_str(ut8 *asciistr, ut8 *in, size_t len_input, charset *custom_charset, size_t len_charset);
```

Comment:
you should not use `charset` here, just `RCharset`.

Reply:
Good idea.